Floofbot/Modules/Fun.cs

```diff
@@ -168,5 +168,172 @@ public async Task RequestCatFact()
             }
             await Context.Channel.SendMessageAsync(fact);
         }
+
+        [Command("minesweeper")]
+        [Summary("Minesweeper minigame")]
+        public async Task Minesweeper([Summary("grid x")]int gridx, [Summary("grid y")]int gridy, [Summary("bomb count")]int bombs)
```

Comment:
Yeah these parameters will have to be renamed width and height imo, rather than just gridx gridy